package.json

```diff
@@ -316,7 +316,7 @@
     "cljs-server-start": "yarn && shadow-cljs start",
     "cljs-server-stop": "yarn && shadow-cljs stop",
     "cljs-server-restart": "yarn && shadow-cljs restart",
-    "cljs-nrepl": "yarn && shadow-cljs node-repl -d cider/cider-nrepl:0.30.0 -d cider/piggieback:0.5.3 -d refactor-nrepl:3.6.0",
+    "cljs-nrepl": "yarn && shadow-cljs node-repl -d cider/cider-nrepl:0.30.0 -d cider/piggieback:0.5.3 -d refactor-nrepl/refactor-nrepl:3.6.0",
```

Comment:
New version of shadow wants you to use qualified deps otherwise prints a warning